general/1_terminal/Max-Kovalchuk/task4_homework.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+$1
+
+jq ".messages[].text" $1
```

Comment:
This is correct and the task is completed.

If you'd like an additional assignment, make it so that the messages that have links or mentions in them are rendered in the same way as the other messages.

Now, they are in an array.